lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,16 @@
+//importing partials
+@import 'nav';
+
+
+
+
+* {
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+
+}
```

Comment:
Don't forget to import _variables.scss, or just delete that file if it's not needed.